tests/run-make/mte-ffi/rmake.rs

```diff
@@ -0,0 +1,39 @@
+// Tests that MTE tags and values stored in the top byte of a pointer (TBI) are
+// preserved across FFI boundaries (C <-> Rust).
+// This test does not require MTE: whilst the test will use MTE if available, if it is not,
+// arbitrary tag bits are set using TBI.
+
+//@ only-aarch64
+//@ only-linux
+//@ only-gnu
```

Comment:
Question: this is intended to be only tested for `aarch64-unknown-linux-gnu`?

Reply:
Yes - I tried `//@ only-aarch64-unknown-linux-gnu` (which I think should work from the compiletest docs, unless I've misunderstood that?) but that kept complaining that it didn't recognise it.

The reason is just that the linker has to be specified when cross-compiling, and I'm not able to easily test other targets.

Reply:
> Yes - I tried `//@ only-aarch64-unknown-linux-gnu` (which I think should work from the compiletest docs, unless I've misunderstood that?) but that kept complaining that it didn't recognise it.

I think you need to add this specifically to a list of known directives (a temporary hack to catch unknown directives while we try to rework compiletest directive parsing) here:

https://github.com/rust-lang/rust/blob/1ddedbaa5919b7b3e70d984660e21e844c615c97/src/tools/compiletest/src/command-list.rs#L4

> The reason is just that the linker has to be specified when cross-compiling, and I'm not able to easily test other targets.

Could you leave that as a comment before/after the ignore? It helps test readers understand *why* it's only the particular target.